File: app/src/main/java/ai/elimu/content_provider/rest/SoundsService.java
package ai.elimu.content_provider.rest;

import java.util.List;

import ai.elimu.model.v2.gson.content.SoundGson;
import retrofit2.Call;
import retrofit2.http.GET;

public interface SoundsService {

    @GET("content/sounds")
    Call<List<SoundGson>> listSounds();
}

File: app/src/main/java/ai/elimu/content_provider/rest/SoundsService.kt
package ai.elimu.content_provider.rest

import ai.elimu.model.v2.gson.content.SoundGson
import retrofit2.Call
import retrofit2.http.GET

interface SoundsService {
    @GET("content/sounds")
    fun listSounds(): Call<List<SoundGson>>
}
